ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -35,7 +35,7 @@ class DemoListViewController: DemoTableViewController {
             Colors.setProvider(provider: provider, for: window)
             FluentUIFramework.initializeAppearance(with: primaryColor, whenContainedInInstancesOf: [type(of: window)])
         } else {
-            FluentUIFramework.initializeAppearance(with: Colors.primary(for: window))
+            FluentUIFramework.initializeAppearance(with: UIColor(colorValue: GlobalTokens.brandColors(.comm80)))
```

Comment:
Does this need a dark mode color? Looks like the comm blue in the asset catalog has both light and dark.

Reply:
Light and dark mode on primary default in the asset catalog are super super close together (both nearly comm90), but I added a dark mode anyway just to be safe.